live-build/misc/migration-scripts/dx_execute

```diff
@@ -85,4 +106,16 @@ MAIN_MENU_LINUX_CMD=$(grep -F 'mainmenu_command[8]' /boot/menu.rc.local |
 	cut -d = -f 2-)
 echo "set menu_timeout_command=$MAIN_MENU_LINUX_CMD" >>/boot/menu.rc.local
 
+# Constants used by the uadmin syscall.
+A_SHUTDOWN=2
+AD_POWEROFF=6
+AD_BOOT=1
+
+# reboot or shutdown
+if $opt_s; then
+	uadmin $A_SHUTDOWN $AD_POWEROFF
```

Comment:
Is there any case of those `uadmin` cases failing? and if yes, is there a way we'll know?